apps/core/scenes/bookingDetail/sectorDetails/segments/SegmentMap.js

```diff
@@ -3,25 +3,96 @@
 import * as React from 'react';
 import { MapView } from '@kiwicom/margarita-map';
 import { StyleSheet } from '@kiwicom/universal-components';
+import { graphql, createFragmentContainer } from '@kiwicom/margarita-relay';
+import { getCenter } from 'geolib';
+import memoize from 'memoize-one';
+import stringify from 'json-stable-stringify';
 
-// TODO: Replace hardcoded values with location coordinates
-const SegmentMap = () => (
-  <MapView
-    style={styles.mapView}
-    initialRegion={{
-      latitude: 37.78825,
-      longitude: -122.4324,
-      latitudeDelta: 0.0922,
-      longitudeDelta: 0.0421,
-    }}
-  />
+import type { SegmentMap as BookingType } from './__generated__/SegmentMap.graphql';
+
+type Props = {|
+  +data: ?BookingType,
+|};
+
+const DEFAULT_PADDING = { top: 40, right: 40, bottom: 40, left: 40 };
+
+const mapCoordinates = memoize(
+  (segmentLocations: ?$PropertyType<BookingType, 'segmentLocations'>) => {
+    const coordinates = segmentLocations ?? [];
+    return coordinates.map(coordinate => ({
+      latitude: coordinate?.lat,
+      longitude: coordinate?.lng,
+    }));
+  },
 );
 
+class SegmentMap extends React.Component<Props> {
+  map: ?React.Ref<typeof MapView>;
+
+  setRef = (ref: ?React.Ref<typeof MapView>) => {
+    this.map = ref;
+  };
+
+  getMappedCoordinates = () => {
+    return mapCoordinates(this.props.data?.segmentLocations);
+  };
+
+  fit = () => {
+    if (this.map != null) {
+      // $FlowExpectedError: fitToCoordinates does exist on map, but flow is confused by .native|.web exports
```

Comment:
I have a suspicion that `react-native-web` flow-typed definitions are the reason behind many of these false positives.

Reply:
I suspect [this one](https://github.com/kiwicom/margarita/blob/master/packages/map/src/MapView.web.js)